### DISCUSSION TITLE
JK BMS has recently started to send only a part of the values

### DISCUSSION THREAD

**Discussion by marcboec** [Louisvdw/dbus-serialbattery#797] [Q&A]
Hello,

I've had a very strange problem for 2 days. Unfortunately, I can't find a solution and hope that someone has already had a similar problem.

I have had a Multiplus 2, a Cerbo GX and a JK BMS running via Bluetooth for about 3 months. Everything worked wonderfully.

For two days now, I no longer receive all the values from the JK BMS in the Cerbo GX. Only the cell voltages and data such as firmware, serial number and parameters are still coming in. But all other data such as SOC, temperature, MOSFET temperature, remaining runtime are no longer transmitted.

If I connect directly to the JK BMS via the app, all values are displayed correctly.

The driver is also running. Version 1.0.20230531 Cerbo GX version 3.01.

I have no idea what the problem is. I would appreciate any help you can give me.

Thanks Marc

**Comment by sanggaard** [comment]
Hello,

I have experienced the exact same problem - only cell voltages and static fields are coming through.  
Voltage, SOC etc. are 0.
On the JK BMS app everything is ok. 

Connection using Bluetooth. Cerbo GX version 3.0
JK_B2A20S20P, hv: V11.XW, sw: V11.21H, version V4.15.6

@Marc: Have you found a solution?

brgds,
Jørn

**Comment by bigblue1735** [comment]
Have someone of you found a solution. Because of connecting issues with the RS485 module i switched to the bluetooth connection and also have this issue about missing values from the BMS.

I changed the BMS password, maybe this is an issue? I also started with verion 0.4X of your famous software. Maybe both of them are running or something else.

Here is my setup:
Cerbo GX with Firmware: 3.11
JK BMS, hv: 11.XW, Firmware: 1.0.20230531

LOG:
There are no ERRORS in the log file, located at: 
/data/log/dbus-blebattery.0/current

**Comment by mr-manuel** [comment]
After positive feedback in https://github.com/Louisvdw/dbus-serialbattery/issues/716 I will update the dev branch, then you can try with that. Meanwhile you can test the current dev branch and change that file manually, if you want.

**Reply by bigblue1735** [reply]
Thank you very much for supporting this software. Hopefully your commit will be approved soon.

**Reply by mr-manuel** [reply]
@bigblue1735 you can also test it. If it works, I will commit instantly.

**Reply by bigblue1735** [reply]
i have done it meanwhile, but it does not work on my side. I changed the jkbms_brn.py file in the bms folder and restart the cerbo. Nothing changed. I also missing the SOC.

**Reply by mr-manuel** [reply]
Can you write me on Discord?

**Reply by bigblue1735** [reply]
hopefully i have done it right. I am not so familiar with discord.

**Comment by mr-manuel** [comment]
The latest `nightly` from the `dev` branch should fix this. Please install it and report back.
